subgraph/query-examples/main.ts

```diff
@@ -0,0 +1,78 @@
+import { ExecutionResult } from 'graphql';
+import {
+  allNameChangesOfTokenZeroDocument,
+  allNameChangesOfTokenZeroQuery,
+  allNameChangesOfTokenZeroTriggeredByUserDocument,
+  allNameChangesOfTokenZeroTriggeredByUserQuery,
+  allNameChangesOfTokenZeroTriggeredByUserQueryVariables,
+  execute,
+  firstFiveMintsByUserDocument,
+  firstFiveMintsByUserQuery,
+  lastFiveMintsDocument,
+  lastFiveMintsQuery,
+  theSecondAndThirdCollectionRoleGrantedsTriggeredByUserDocument,
+  theSecondAndThirdCollectionRoleGrantedsTriggeredByUserQuery,
+  theThreeNewestBuildsThatWereNotTriggeredByUserDocument,
+  theThreeNewestBuildsThatWereNotTriggeredByUserQuery,
+} from './.graphclient';
+
+function main() {
+  execute(firstFiveMintsByUserDocument, {}).then(
+    (result: ExecutionResult<firstFiveMintsByUserQuery>) => {
+      console.log('\nFirst five mints by the user:');
+      console.log(result.data?.transfers);
+    }
+  );
+
```

Comment:
Just a hint, there is no changes needed here:
As this is just a function being called, we don't need to have a function is just use the module scope to run all. This would be cool if the `main` function was an async one, so the code could be rewriten like:
```suggestion
function async main() {
 const result = await execute(firstFiveMintsByUserDocument, {});
 console.log('\nFirst five mints by the user:');
 console.log(result.data?.transfers);

```

Reply:
Initially, that's how I wrote the script but I got Promise errors when I tried to run the script.

Reply:
Let's quickly try to fix this and include but if its something major we can push to next sprint.